cdk/src/functions/living-docs-dashboard-generator.ts

```diff
@@ -30,13 +32,13 @@ export async function handler(_event: any, _context: any) {
   );
 }
 
-async function isRootMailConfigured() {
-  const rootMailReadyAlarm = await cloudwatchClient.send(
-    new DescribeAlarmsCommand({
-      AlarmNames: ['superwerker-RootMailReady'],
+async function isRootMailConfigured(propagationParamName: string) {
```

Comment:
Is there a reason for switching from CW Alarm to SSM Parameter?

Reply:
In the old rootmail template, it had a wait condition that gets triggered by CW alarm for the DNS propagation. In CDK we use Provider isCompleteHandler() that takes care of that & we don't need the CW alarm. 

We still need a way to communicate the status with living-docs so I use SSM param for that.